jit.js

```diff
@@ -198,13 +380,31 @@ to single-step.
             "    inst = rcvr.pointers,\n",
             "    temp = vm.homeContext.pointers,\n",
             "    lit = vm.method.pointers,\n",
-            "    bytecodes = 0 - vm.pc;\n",
-            "while (true) switch (vm.pc) {\n"
-        );
+            "    bytecodes = 0 - vm.pc;\n");
+
+        this.labelBootstrapStart = this.source.length;
+        this.source.push("while (true) switch (vm.pc) {\n");
+        
         this.done = false;
         while (!this.done) {
-            var byte = method.bytes[this.pc++],
-                byte2 = 0;
+            var byte = method.bytes[this.pc++];
```

Comment:
Move this line to after the peep-hole code, then you don't need to use ```pc - 1``` everywhere.